### PR TITLE
Issue/date overload minus

### LINKED ISSUE #149: Date should overload minus operator
DateTime and DateTimeTz overload both plus and minus operators, but for some reason Date only overloads the plus operator. This means you have to do things like this:

    val yesterday = today + (-1).days

### PR DESCRIPTION
**Pull request by IgnacioCarrionN**
Fix #149 

Add overload operator for Date minus.
Add tests for new overload operators.

### CODE REVIEW

File: klock/src/commonMain/kotlin/com/soywiz/klock/wrapped/WDate.kt [hunk 1]
```diff
@@ -63,6 +63,11 @@ data class WDate(val value: Date) : Comparable<WDate>, Serializable {
     operator fun plus(time: WDateTimeSpan) = (this.value + time.value).wrapped
     operator fun plus(time: WTime) = (this.value + time.value).wrapped
 
+    operator fun minus(time: WTimeSpan) = (this.value - time.value).wrapped
+    operator fun minus(time:WMonthSpan) = (this.value - time.value).wrapped
```
Comment (soywiz) [inline klock/src/commonMain/kotlin/com/soywiz/klock/wrapped/WDate.kt:67]: Nitpicking: space after `:`

File: klock/src/commonTest/kotlin/com/soywiz/klock/DateTest.kt [hunk 2]
```diff
@@ -39,5 +43,8 @@ class DateTest {
 		}
 
 		assertEquals("Wed, 18 Sep 2019 00:00:00 UTC", (Date(2019, Month.September, 17) + Time(24)).toString())
+        assertEquals("Tue, 17 Sep 2019 00:00:00 UTC", (Date(2019, Month.September, 18) - Time(24)).toString())
```
Comment (soywiz) [inline klock/src/commonTest/kotlin/com/soywiz/klock/DateTest.kt:46]: I know that with the plus the tests were a bit lacky, but can you add a few more tests including minutes, seconds / milliseconds, just to be sure it is working properly?

File: klock/src/commonMain/kotlin/com/soywiz/klock/wrapped/WDate.kt (hunk 1 above)
Comment (IgnacioCarrionN) [reply klock/src/commonMain/kotlin/com/soywiz/klock/wrapped/WDate.kt]: Done

File: klock/src/commonTest/kotlin/com/soywiz/klock/DateTest.kt (hunk 2 above)
Comment (IgnacioCarrionN) [reply klock/src/commonTest/kotlin/com/soywiz/klock/DateTest.kt:47]: Done

**Review by soywiz — APPROVED** [review]
Nice. Thanks!

**Pull request merged**